.github/workflows/mirror-images.yaml

```diff
@@ -19,6 +19,8 @@ jobs:
         run: |
           for IMAGE in \
             library/golang:1.16 \
+            library/golang:1.17 \
+            library/golang:1.18 \
             library/maven:3-jdk-8-openj9 \
             library/node:12 \
```

Comment:
may you quickly also add node:14 and node:16 for the same reason ?

Reply:
sure, of course